src/main/java/client/Client.java

```diff
@@ -0,0 +1,29 @@
+package main.java.client;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+
+public class Client {
+
+    private static final int PORT = 8080;
+    private static final String HOST = "127.0.0.1";
+
+
+    public static void main(String[] args) {
+        try {
+            Socket socket = new Socket(HOST, PORT);
+            BufferedReader br = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            String str = null;
+            while ((str = br.readLine()) != null ){
+                System.out.println(str);
+            }
+
+            socket.close();
+
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=1&issues=AWQTU_q-bXXTsx7O43dY&open=AWQTU_q-bXXTsx7O43dY)

[ISSUE_KEY]: AWQTU_q-bXXTsx7O43dY